packages/now-cli/src/util/dev/server.ts

```diff
@@ -883,17 +884,22 @@ export default class DevServer {
       ops.push(shutdownBuilder(match, this.output));
     }
 
-    ops.push(
-      new Promise(resolve => {
-        if (!this.devProcess) {
-          resolve();
-          return;
-        }
-
-        this.devProcess.on('exit', () => resolve());
-        process.kill(this.devProcess.pid, exitCode);
-      })
-    );
+    if (devProcess) {
+      ops.push(
+        new Promise((resolve, reject) => {
+          devProcess.once('exit', () => resolve());
+          try {
+            process.kill(devProcess.pid);
```

Comment:
```suggestion
            process.kill(devProcess.pid, exitCode);
```

 I think you're missing exitCode here.

Reply:
I removed it on purpose. [You don't pass in an exit code to `process.kill()`, you pass in a signal (string or number)](https://nodejs.org/api/process.html#process_process_kill_pid_signal). The default kill signal of `"SIGTERM"` seems appropriate to me here.